src/core_modules/capture-core/components/FormFields/New/CategoryOptions/CategoryOptions.component.js

```diff
@@ -0,0 +1,108 @@
+// @flow
```

Comment:
If the category components for Scheduling have to be different than the other ones, let's put these files in the Scheduling folder.